Source/Krypton Components/Krypton.Ribbon/Controller/GalleryButtonController.cs

```diff
@@ -25,7 +25,8 @@ internal class GalleryButtonController : GlobalId,
         private bool _pressed;
         private bool _mouseOver;
         private NeedPaintHandler? _needPaint;
-        private readonly Timer _repeatTimer;
+        private readonly Timer? _repeatTimer;
```

Comment:
Why is this nullable ?
If it is set to null in Dispose, then that's okay !

Reply:
It crashed in testing. Maybe use `!`?

Reply:
That means something else is wrong.

Reply:
Now fixed, the timer wasn't disposed of